src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -604,7 +604,8 @@ BinaryExpressionSyntax or
                 AssignmentExpressionSyntax or
                 QueryClauseSyntax or
                 SelectOrGroupClauseSyntax or
-                CheckedExpressionSyntax)
+                CheckedExpressionSyntax or 
```

Comment:
@dotnet/roslyn-compiler does this need compiler reviews for SharedUtilitiesAndExtensions?

Reply:
Based on the history in this file, I don't think so. It doesn't look like we've reviewed previous PRs touching this file.